conans/test/functional/toolchains/gnu/autotools/test_ios.py

```diff
@@ -74,9 +74,9 @@ def build(self):
     configure_args = conanbuild["configure_args"]
     make_args = conanbuild["make_args"]
     autoreconf_args = conanbuild["autoreconf_args"]
-    assert configure_args == "'--prefix=/' '--bindir=${prefix}/bin' '--sbindir=${prefix}/bin' " \
+    assert configure_args == "--prefix=/ '--bindir=${prefix}/bin' '--sbindir=${prefix}/bin' " \
                              "'--libdir=${prefix}/lib' '--includedir=${prefix}/include' " \
                              "'--oldincludedir=${prefix}/include' '--datarootdir=${prefix}/res' " \
-                             "'--host=aarch64-apple-ios' '--build=x86_64-apple-darwin'"
+                             "--host=aarch64-apple-ios --build=x86_64-apple-darwin"
```

Comment:
It is not necessary to put quotes if there is no unsafe characters like the `$` so these are not quoted